apps/meteor/app/livechat/server/api/lib/users.ts

```diff
@@ -38,8 +48,18 @@ async function findUsers({ role, text, pagination: { offset, count, sort } }) {
 		total,
 	};
 }
-export async function findAgents({ text, pagination: { offset, count, sort } }) {
-	return findUsers({
+export async function findAgents({
+	text,
+	pagination: { offset, count, sort },
+}: {
+	text: string | undefined;
+	pagination: IPaginationOptions & {
```

Comment:
Create a helper type "WithSort<>" and pass IPaginationOptions, or create an extended type 👀 (so you don't repeat & { sort }...)